introduction.md

```diff
@@ -6,27 +6,101 @@ title: Introduction
 # Introduction
 
 ## What is Foreman?
-Foreman is an open source project that helps system administrators manage servers throughout their lifecycle,
-from provisioning and configuration to orchestration and monitoring.
-Using Puppet, Chef, Salt, Ansible and Foreman's smart proxy architecture, you can easily automate repetitive tasks,
-quickly deploy applications, and proactively manage change, both on-premise with VMs and bare-metal or in the cloud.
 
-Foreman provides comprehensive, interaction facilities including a web frontend, CLI and RESTful API
-which enables you to build higher level business logic on top of a solid foundation.
+Foreman is an open source project that helps system administrators manage servers throughout their lifecycle, from provisioning and configuration to orchestration and monitoring. Provisioning support gives you easy control of setting up new servers, and using configuration management (Puppet, Ansible, Chef and Salt are supported), you can easily automate repetitive tasks. With Foreman, you can quickly deploy applications, and proactively manage change, both on-premise with VMs and bare-metal or in the cloud. Foreman scales well to multiple locations (offices, data centres, etc) and multiple organisations, allowing you to grow without losing your single source of infrastructure truth.
```

Comment:
s/organisations/organizations/

Reply:
That's British form. We have a lot of it on our site already.

Reply:
It's still strange, since in the app we're using `organization` everywhere, But leaving it up to you